server/src/main/java/org/elasticsearch/cluster/coordination/Coordinator.java

```diff
@@ -647,7 +647,11 @@ protected void doStart() {
             coordinationState.set(new CoordinationState(settings, getLocalNode(), persistedState));
             peerFinder.setCurrentTerm(getCurrentTerm());
             configuredHostsResolver.start();
-            VotingConfiguration votingConfiguration = coordinationState.get().getLastAcceptedState().getLastCommittedConfiguration();
+            final ClusterState lastAcceptedState = coordinationState.get().getLastAcceptedState();
+            if (lastAcceptedState.metaData().clusterUUIDCommitted()) {
+                logger.info("cluster UUID [{}]", lastAcceptedState.metaData().clusterUUID());
```

Comment:
should we also log the cluster name here?

Reply:
Interesting. I thought we made a bit more noise when mixing cluster names but it turns out we don't. Since its lifecycle is more related to `node.name` I thought it'd make most sense to log it in the constructor of `Node`.